modules/administration-guide/partials/proc_configuring-che-with-custom-resources-to-use-amazon-s3.adoc

```diff
@@ -0,0 +1,62 @@
+[id="configuring-{prod-id-short}-with-custom-resources-to-use-amazon-s3_{context}"]
+= Configuring {prod-short} with custom resources to use Amazon S3
+
+To configure {prod-short} to use Amazon S3:
+
+.Prerequisites
+
+* Configured Amazon S3. See xref:supported-restic-compatible-backup-servers.adoc[].
+
+.Procedure
+
+. Create a secret containing the `repo-password` key with a Base64 encoded password:
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backup-encryption-password-secret
+  namespace: eclipse-che
+type: Opaque
+stringData:
+  repo-password: __<base64_encoded_password>__
+----
++
+({prod-short} Operator will set up this password for the backup repository that {prod-short} Operator will create from this custom object on the backup server.)
++
+include::snip_warning-about-losing-backup-repo-password.adoc[]
+
+. Create a secret (for example, `name: aws-user-credentials-secret`) containing:
+.. `awsAccessKeyId` key with the Base64 encoded AWS access key ID of the user
+.. `awsSecretAccessKey` key with the Base64 encoded AWS secret access key of the user
+
+. Create the `CheBackupServerConfiguration` custom object:
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+apiVersion: org.eclipse.che/v1
+kind: CheBackupServerConfiguration
+metadata:
+  name: backup-server-configuration
+  namespace: {prod-namespace}
+spec: <1>
+  awss3:
+    protocol: https <2>
+    hostname: s3.amazonaws.com <3>
+    port: 1234 <4>
+    repositoryPath: {prod-id}-backups <5>
+    repositoryPasswordSecretRef: backup-encryption-password-secret <6>
+    awsAccessKeySecretRef: aws-user-credentials-secret <7>
+----
+<1> The `spec` property of this custom object must only have one section configured.
+<2> Optional property that specifies the protocol to be used: `https` is the default value; `http` is a permitted value.
+<3> Optional property that specifies the S3 hostname. The default value is `s3.amazonaws.com`.
+<4> Optional property that specifies the port on which the backup server is running.
```

Comment:
Is there a default value? If so, what is it?

Reply:
I'm asking the SME. Thank you.